src/views/components-wallet/TheWalletTopRightText.vue

```diff
@@ -0,0 +1,37 @@
+<template>
```

Comment:
Please implement the following: 
- move the component to core/components
- rename the component to AppButtonBalance


views folder is disignanted to layouts and components that are specific to the layout.  please refer to the vue guide on how to name components